releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -2,6 +2,14 @@
 Azure App Configuration Push extension for Azure DevOps pipeline can be installed from the [Azure DevOps marketplace](https://marketplace.visualstudio.com/items?itemName=AzureAppConfiguration.azure-app-configuration-task-push). The version information can be found in the console when the task is executed:
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
+### v4.4.0 - November, 15 2022
+* Added ImportMode option support for KVSet profile.
+* Added Strict support for KVSet profile. With the added support of Strict for KVSet profile, when Strict is set to true any key-values in the store that are not included in the configuration file will be deleted.

```

Comment:
"With the added support of Strict for KVSet profile, when"

It kind of repeats the previous sentence. Can you change that to "When using the KVSet profile and setting `Strict` to true ..."